### ISSUE TITLE
Lifecycle Statuses for Assessments, and Learning Opportunities

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/Schema-Development#675]
Per recent internal discussions:

We need to enable indicating the status of organizations, assessments, and learning opportunities - similar to what we do with [Credential Status](https://credreg.net/ctdl/terms/CredentialStatus) but written generically enough to apply to other types of entities.

I propose:

**Property:**
> **URI:** lifeCycleStatusType
> **Label:** Life Cycle Status Type
> **Definition:** Type of status of the resource in terms of its stage in a life cycle; select from an enumeration of such types.
> **Domain:** ceterms:AssessmentProfile, ceterms:LearningOpportunityProfile, ceterms:CredentialOrganization, ceterms:QACredentialOrganization, ceterms:TransferValueProfile (#683)
> **Range:** ceterms:CredentialAlignmentObject
> **Target Concept Scheme:** ceterms:StatusCategory
> **Equivalent Property:** schema:creativeWorkStatus

**Concept Scheme:**
> **URI:** ceterms:LifeCycleStatusCategory
> **Label:** Life Cycle Status Category
> **Definition:** Types of status of a resource in terms of its stage in a life cycle.

**Concepts:**
> **URI:** lifeCycle:Developing
> **Label:** Developing
> **Definition:** Resource is under development and may or may not become active, current, ongoing, offered, operational, or available.
> **In Scheme:** ceterms:StatusCategory
> **Close Match:** publicationStatus:Draft

> **URI:** lifeCycle:Active
> **Label:** Active
> **Definition:** Resource is active, current, ongoing, offered, operational, or available.
> **In Scheme:** ceterms:StatusCategory
> **Close Match:** credentialStat:Active
> **Close Match:** publicationStatus:Published

> **URI:** lifeCycle:Suspended
> **Label:** Suspended
> **Definition:** Resource is temporarily not active, current, ongoing, offered, operational, or available.
> **In Scheme:** ceterms:StatusCategory
> **Close Match:** credentialStat:Suspended

> **URI:** lifeCycle:Ceased
> **Label:** Ceased
> **Definition:** Resource is permanently no longer active, current, ongoing, offered, operational, or available.
> **In Scheme:** ceterms:StatusCategory
> **Close Match:** credentialStat:Deprecated
> **Close Match:** publicationStatus:Deprecated

**Other schema stuff:**
**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** skos:closeMatch
> **Object:** lifeCycle:Active

> **Subject:** credentialStat:Suspended
> **Predicate:** skos:closeMatch
> **Object:** lifeCycle:Suspended

> **Subject:** credentialStat:Deprecated
> **Predicate:** skos:closeMatch
> **Object:** lifeCycle:Ceased

> **Subject:** publicationStatus:Draft
> **Predicate:** skos:closeMatch
> **Object:** lifeCycle:Developing

> **Subject:** publicationStatus:Published
> **Predicate:** skos:closeMatch
> **Object:** lifeCycle:Active

> **Subject:** publicationStatus:Deprecated
> **Predicate:** skos:closeMatch
> **Object:** lifeCycle:Ceased

I'm not sure whether skos:closeMatch or skos:related is more appropriate here - the meanings of the terms are basically the same, but their scopes are different (one set applying only to credentials and the other to any kind of resource).

I don't think we need an equivalent term for credentialStat:Probationary, since that notion doesn't really apply to other kinds of resources.

Also, maybe "Resource Status" would be preferable to "Status Category"?

**Comment by philbarker** [comment]
related: https://schema.org/creativeWorkStatus 

I think it is worth adding to the definition that this property relates to status "in terms of its stage in a lifecycle".

FWIW, I prefer StatusCategory to ResourceStatus because it is a category, but I think your convention until now has been ResourceStatus

**Comment by siuc-nate** [comment]
>I think it is worth adding to the definition that this property relates to status "in terms of its stage in a lifecycle".

Works for me. I've updated the definitions for the property and the concept scheme in the first post.

**Comment by stuartasutton** [comment]
works for me.

**Comment by jeannekitchens** [comment]
Before we make any updates, lets look at the organization and any other relevant mappings just completed with CEDS and other data standards and ensure we're in alignment.

**Comment by siuc-nate** [comment]
Per our 4-22-2020 meeting:
This, or aspects of it (like the concept scheme), may be useful for providing statuses for competencies.

**Comment by siuc-nate** [comment]
This came up again in our ProPath meeting today.

**Comment by jeannekitchens** [comment]
Moving this issue up to an April milestone.  We have another state, Minnesota,  that will require these statuses.

1. A few of the institutions are currently pursuing a merger, so I anticipate reorganization of credentials offered and the institution(s) that the programs are housed within.

2. Schools that have a license as a career school sometimes change status. We’ve had a handful of training providers close or transition this last year because a) the instructors were not teaching the content safely such that the state license that allows the training provider to operate legally as a training provider was revoked, or b) the owner/teaching staff died, and the inheritor chose to close the school, or c) the economic conditions of the industry changed such that the organization pursued selling goods and services and is no longer providing training/education.

**Comment by jeannekitchens** [comment]
This proposal has been updated to be pertinent only to learning opps and assessments that have an end of life.  The prefernce is to utilize the existing Credential Status concept scheme for https://credreg.net/ctdl/terms#CredentialStatus

1. Rename it Status, rather than Credential Status
2. Expand the defintion to be applicable to a resource (e.g., credential, learning opp, assessment).  
3. Add a comment to provide examples as described below. 

Background.  State are publishing thousands of credentials and learning opportunities.  Learning opportunities can have an end of life.  Those learning opps could be for credentials that continue to be offered.  For example, a learning opportunity may prepare for a credential offered by another organization (e.g., certification offered by a national certification body).  

Assessments can also have an end of life.  They may be for a credential that has changed and thereby the assessment has been substantially revised and is a new version.  

Regardless of the reason for no longer offering the learning opportunity or assessment, either scenario occurs irrespective of the relationship to a credential.   

There is an immediate technical need to address this.  We have states that need to change the status rather than delete learning opps and assessments.

**Comment by philbarker** [comment]
1) is the vocabulary set in stone? It's just that "Deprecated" seems a strong word for "no longer offered". I think we need to be mindful that while we do not describe credentials held by individuals we do encourage organizations who are doing so to use CTDL where appropriate. I'm concerned that in that context "Deprecated" might be read as "this credential is no londer valid".

2) related: [schema:creativeWorkStatus](https://schema.org/creativeWorkStatus)

**Comment by jeannekitchens** [comment]
@philbarker we have had that very conversation, the credential, learning opp or assessment are no longer offered.  It has to impact on the validity of the credentials issued to people or the validity of completed courses or assessments.  This concept scheme has been in use for quite awhile and thousands of credentials are published to the Registry using a status.  If there should be a usage note or other clarifications, let's get those proposals down.  We should do this in a way that does not deprecate the concept scheme or concepts.

**Comment by philbarker** [comment]
@jeannekitchens that answers my question, we should consider it set in stone.

**Comment by stuartasutton** [comment]
Before doing anything here, we need to take up the discussion of--CredentialingActions that is defined as an "[a]ction taken by an agent affecting the status of an object entity.” We have a discussion document in process: https://docs.google.com/document/d/1yyQV27l_1pW4xgcXqYRitSFeX6mFXeeydHMzyInyzUc/edit#

**Comment by siuc-nate** [comment]
There may be an immediate technical need, but let's not rush things that may come back to bite us later - we don't want to end up being stuck with something that isn't good enough or has some unforeseen flaw because we didn't think it through all the way.

If we are going to go with updating the existing concept scheme (which I'm not entirely opposed to, but hesitant given how strongly defined it is relative to credentials), what would the revised definitions look like?  My guess would be:

- Active:
  - Activity with the resource is ongoing.
- Deprecated
  - Activity with the resource has ceased.
- Probationary
  - Activity with the resource is provisional or temporary; frequently pending further assessment of qualification.
- Suspended
  - Activity with the resource is not currently ongoing, but may resume.

**Comment by jeannekitchens** [comment]
StatusType - 

Status - 
   - Credential Status Type - No Change
       Active: Activity with the resource is ongoing.
       Deprecated: Activity with the resource has ceased.
       Probationary: Activity with the resource is provisional or temporary; frequently pending further assessment of qualification.
       Suspended: Activity with the resource is not currently ongoing, but may resume.

**Comment by siuc-nate** [comment]
**Create:**
> **URI:** ceterms:statusType
> **Label:** Status Type
> **Definition:** Type of status of the resource; select from an enumeration of such types.
> **Domain:** ceterms:AssessmentProfile, ceterms:LearningOpportunityProfile
> **Range:** ceterms:CredentialAlignmentObject
> **Target Scheme:** ceterms:StatusCategory

**Create:**
> **URI:** ceterms:StatusCategory
> **Label:** Status Category
> **Definition:** Types of statuses of a resource.

**Add:**
> **Subject:** ceterms:credentialStatusType
> **Predicate:** rdfs:subPropertyOf
> **Object:** ceterms:statusType

**Add:**
> **Subject:** ceterms:CredentialStatus
> **Predicate:** rdfs:subClassOf
> **Object:** ceterms:StatusCategory

**Add:**
> **Subject:** credentialStat:Active, credentialStat:Deprecated, credentialStat:Probationary, credentialStat:Suspended
> **Predicate:** skos:inScheme
> **Object:** ceterms:CredentialStatus

**Remove:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential are ongoing.

**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Resource is currently offered.

**Remove:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential have ceased.

**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Resource has ceased to be offered.

**Remove:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Credential award is provisional or temporary; frequently pending further assessment of qualification.

**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Resource is offered provisionally or temporarily; frequently pending further assessment of qualification.

**Remove:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential are not available, but may resume.

**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Resource is not currently offered, but may resume.

**Comment by siuc-nate** [comment]
In practice, all of these options would be structurally identical (via CredentialAlignmentObject -> targetNode etc) so rather than a cluttered list of JSON examples, I'll simplify what the options look like for each flavor of status with the following list.

Resource that is in the works
---
This borrows the notion of a pre-release/draft state from CTDL-ASN and explores its use in other objects.

- Current Schema:
  - Organizations: N/A
  - Credentials: N/A
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: ceterms:publicationStatusType - publicationStatus:Draft
  - Concept Schemes: ceterms:publicationStatusType - publicationStatus:Draft

- Modification Proposal:
  - Organizations: N/A
  - Credentials: N/A
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: ceterms:publicationStatusType - publicationStatus:Draft
  - Concept Schemes: ceterms:publicationStatusType - publicationStatus:Draft

- New Terms Proposal:
  - Organizations: ceterms:lifeCycleStatusType - lifeCycle:Developing
  - Credentials: ceterms:lifeCycleStatusType - lifeCycle:Developing
  - Assessments: ceterms:lifeCycleStatusType - lifeCycle:Developing
  - Learning Opportunities: ceterms:lifeCycleStatusType - lifeCycle:Developing
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Draft
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Draft

Resource that is "Probationary"
---
It's not quite clear how this would apply outside of the scope of credentials, but its use is included here for completeness.

- Current Schema:
  - Organizations: N/A
  - Credentials: ceterms:credentialStatusType - credentialStat:Probationary
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: N/A
  - Concept Schemes: N/A

- Modification Proposal:
  - Organizations: ceterms:statusType - credentialStat:Probationary
  - Credentials: ceterms:credentialStatusType - credentialStat:Probationary
  - Assessments: ceterms:statusType - credentialStat:Probationary
  - Learning Opportunities: ceterms:statusType - credentialStat:Probationary
  - Competency Frameworks: N/A
  - Concept Schemes: N/A

- New Terms Proposal:
  - Organizations: N/A
  - Credentials: ceterms:credentialStatusType - credentialStat:Probationary
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: N/A
  - Concept Schemes: N/A

Resource that is currently available
---
Available now.

- Current Schema:
  - Organizations: N/A
  - Credentials: ceterms:credentialStatusType - credentialStat:Active
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Active
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Active

- Modification Proposal:
  - Organizations: ceterms:statusType - credentialStat:Active
  - Credentials: credentialStatusType - credentialStat:Active
  - Assessments: ceterms:statusType - credentialStat:Active
  - Learning Opportunities: ceterms:statusType - credentialStat:Active
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Active
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Active

- New Terms Proposal:
  - Organizations: ceterms:lifeCycleStatusType - lifeCycle:Active
  - Credentials: ceterms:credentialStatusType - credentialStat:Active
  - Assessments: ceterms:lifeCycleStatusType - lifeCycle:Active
  - Learning Opportunities: ceterms:lifeCycleStatusType - lifeCycle:Active
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Published
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Published

Resource that is temporarily halted
---
Not available now, but may be later.

- Current Schema:
  - Organizations: N/A
  - Credentials: ceterms:credentialStatusType - credentialStat:Suspended
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: N/A
  - Concept Schemes: N/A

- Modification Proposal:
  - Organizations: ceterms:statusType - credentialStat:Suspended
  - Credentials: ceterms:credentialStatusType - credentialStat:Suspended
  - Assessments: ceterms:statusType - credentialStat:Suspended
  - Learning Opportunities: ceterms:statusType - credentialStat:Suspended
  - Competency Frameworks: N/A
  - Concept Schemes: N/A

- New Terms Proposal:
  - Organizations: ceterms:lifeCycleStatusType - lifeCycle:Suspended
  - Credentials: ceterms:credentialStatusType - credentialStat:Active
  - Assessments: ceterms:lifeCycleStatusType - lifeCycle:Suspended
  - Learning Opportunities: ceterms:lifeCycleStatusType - lifeCycle:Suspended
  - Competency Frameworks: N/A
  - Concept Schemes: N/A

Resource that is over and done with
---
It may still technically exist, but is not active.

- Current Schema:
  - Organizations: N/A
  - Credentials: ceterms:credentialStatusType - credentialStat:Deprecated
  - Assessments: N/A
  - Learning Opportunities: N/A
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Deprecated
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Deprecated

- Modification Proposal:
  - Organizations: ceterms:statusType - credentialStat:Deprecated
  - Credentials: ceterms:credentialStatusType - credentialStat:Deprecated
  - Assessments: ceterms:statusType - credentialStat:Deprecated
  - Learning Opportunities: ceterms:statusType - credentialStat:Deprecated
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Deprecated
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Deprecated

- New Terms Proposal:
  - Organizations: ceterms:lifeCycleStatusType - lifeCycle:Ceased
  - Credentials: ceterms:credentialStatusType - credentialStat:Deprecated
  - Assessments: ceterms:lifeCycleStatusType - lifeCycle:Ceased
  - Learning Opportunities: ceterms:lifeCycleStatusType - lifeCycle:Ceased
  - Competency Frameworks: ceasn:publicationStatusType - publicationStatus:Deprecated
  - Concept Schemes: ceasn:publicationStatusType - publicationStatus:Deprecated

**Comment by siuc-nate** [comment]
As an alternative, we should take a step back and consider whether any indicator of status beyond "it's available" or "it's not" is necessary for Assessments, Learning Opportunities, and the like.  If not, then instead of any changes to status vocabularies (at least at this time), we should consider whether the dateEffective/expirationDate properties, which are already on Assessment and Learning Opportunity, are sufficient.  

In other words, if the current date is before the dateEffective or after the expirationDate, it is no longer available; otherwise, assume that it is.  The question of status for Organizations may be similarly handled by [dissolutionDate](https://github.com/CredentialEngine/Schema-Development/issues/770#issuecomment-843163633)

**Comment by jeannekitchens** [comment]
Date and Status go together, they complement each other.  Don't agree it's one or the other.

**Comment by siuc-nate** [comment]
If you have the dates, you don't need the statuses.  For example:

Learning Opportunity w/ dates:
```
{
	"@type": "ceterms:LearningOpportunityProfile",
	"ceterms:name": { "en": "Electronics 101" },
	"ceterms:dateEffective": "1985-01-01",
	"ceterms:expirationDate": "1990-01-01"
}
``
That actually gives you more information than a status alone (in either implementation):
```
{
	"@type": "ceterms:LearningOpportunityProfile",
	"ceterms:name": { "en": "Electronics 101" },
	"ceterms:statusType": {
		"ceterms:targetNode": "credentialStat:Deprecated",
		"ceterms:targetNodeName": { "en": "Deprecated" }
	}
}

{
	"@type": "ceterms:LearningOpportunityProfile",
	"ceterms:name": { "en": "Electronics 101" },
	"ceterms:lifeCycleStatusType": {
		"ceterms:targetNode": "lifeCycle:Ceased",
		"ceterms:targetNodeName": { "en": "Ceased" }
	}
}
```

You could combine them, but it'd just be redundant:
```
{
	"@type": "ceterms:LearningOpportunityProfile",
	"ceterms:name": { "en": "Electronics 101" },
	"ceterms:dateEffective": "1985-01-01",
	"ceterms:expirationDate": "1990-01-01",,
	"ceterms:statusType": {
		"ceterms:targetNode": "credentialStat:Deprecated",
		"ceterms:targetNodeName": { "en": "Deprecated" }
	}
}

{
	"@type": "ceterms:LearningOpportunityProfile",
	"ceterms:name": { "en": "Electronics 101" },
	"ceterms:dateEffective": "1985-01-01",
	"ceterms:expirationDate": "1990-01-01",
	"ceterms:lifeCycleStatusType": {
		"ceterms:targetNode": "lifeCycle:Ceased",
		"ceterms:targetNodeName": { "en": "Ceased" }
	}
}
```
I don't think we should do that unless there's a specific need/demand/justification for it.

Organization works the same way, but with the existing [foundingDate](https://credreg.net/ctdl/terms/foundingDate) and proposed [dissolutionDate](https://github.com/CredentialEngine/Schema-Development/issues/770#issuecomment-843163633) properties:
```
{
	"@type": "ceterms:CredentialOrganization",
	"ceterms:name": { "en": "Vendor" },
	"ceterms:foundingDate": "1980",
	"ceterms:dissolutionDate": "2002-03-18"
}
```
is more helpful than
```
{
	"@type": "ceterms:CredentialOrganization",
	"ceterms:name": { "en": "Vendor" },
	"ceterms:statusType": {
		"ceterms:targetNode": "credentialStat:Deprecated",
		"ceterms:targetNodeName": { "en": "Deprecated" }
	}
}

{
	"@type": "ceterms:CredentialOrganization",
	"ceterms:name": { "en": "Vendor" },
	"ceterms:lifeCycleStatusType": {
		"ceterms:targetNode": "lifeCycle:Ceased",
		"ceterms:targetNodeName": { "en": "Ceased" }
	}
}
```
and renders this redundant:
```
{
	"@type": "ceterms:CredentialOrganization",
	"ceterms:name": { "en": "Vendor" },
	"ceterms:foundingDate": "1980",
	"ceterms:dissolutionDate": "2002-03-18"
	"ceterms:statusType": {
		"ceterms:targetNode": "credentialStat:Deprecated",
		"ceterms:targetNodeName": { "en": "Deprecated" }
	}
}

{
	"@type": "ceterms:CredentialOrganization",
	"ceterms:name": { "en": "Vendor" },
	"ceterms:foundingDate": "1980",
	"ceterms:dissolutionDate": "2002-03-18"
	"ceterms:lifeCycleStatusType": {
		"ceterms:targetNode": "lifeCycle:Ceased",
		"ceterms:targetNodeName": { "en": "Ceased" }
	}
}
```

**Comment by philbarker** [comment]
@jeannekitchens @siuc-nate @stuartasutton @mparsons-ce I'm trying to write this in to the proposal for course types as Jeanne requested in the agenda for our last meeting, but it doesn't look fully baked.

On Nate's step back to examine whether lifecycle status is redundant because we can use `dateEffective` or `expirationDate`, I think the answer is no, lifecycle status can provide additonal information such as whether a resource is suspended.

But there are two proposals in the thread above ([original](https://github.com/CredentialEngine/Schema-Development/issues/675#issue-600365718) and [counter](https://github.com/CredentialEngine/Schema-Development/issues/675#issuecomment-843393462)) and I have no rcollection of what motivated the counter or where our discussion of the pros and cons of either landed. The original could apply to more types of resource and is more specific about being a lifecycle. The counter is more specific to learning opportunities, I guess it is meant to refer to life cycle but it just says "status" in the name and definition, so as drafted it could be used to indicate whether a Learning Opportunity is "important" i.e. of high status, or "accredited" or a whole load of other status-type concepts. It does, however, have some useful additions to the original especially where it declares the property `statusType` as a super-property of `credentialStatusType` (curiously this would be more useful in the broader scoping of the original proposal). 
There are also differences in the concept schemes proposed: the orginal proposal has Developing Active Suspended Ceased, I think the counter is intended to reuse the existing ceterms:CredentialStatus concept scheme, but it creates  StatusCategory as a sub class of CredentialStatus (I'm really not sold on something be a sub class of a skos:ConceptScheme), and has an addition

> **Add:**
> 
> > Subject: credentialStat:Active, credentialStat:Deprecated, credentialStat:Probationary, credentialStat:Suspended
> > Predicate: skos:inScheme
> > Object: ceterms:CredentialStatus

but these concepts are already members of the CredentialStatus concept scheme, so maybe something else was intended?


## Proposal

_anyway_ synthesizing the bits I like from each proposal:

### Add a new lifeCycle Property
A general purpose property for indicating the stage of development of a resource, of which we can have sub properties; stick with the existing credential status type.

**Create**
> **URI:** lifeCycleStatusType
> **Label:** Life Cycle Status Type
> **Definition:** Type of state of the resource in terms of its stage in a life cycle; select from an enumeration of such types.
> **Domain:** ceterms:AssessmentProfile, ceterms:LearningOpportunityProfile, ceterms:CredentialOrganization, ceterms:QACredentialOrganization, ceterms:TransferValueProfile (#683)
> **Range:** ceterms:CredentialAlignmentObject
> **Target Concept Scheme:** ceterms:CredentialStatus
> **Equivalent Property:** schema:creativeWorkStatus

**Add**

> **Subject:** ceterms:credentialStatusType
> **Predicate:** rdfs:subPropertyOf
> **Object:** ceterms:statusType

### Adjust definitions of CredentialStatusType concepts scheme accordingly

**Remove**

> **Subject** ceterms:CredentialStatus
> **Predicate** rdfs:comment 
> **Object** "Types of applicability statuses of a credential."

**Add**

> **Subject** ceterms:CredentialStatus
> **Predicate** rdfs:comment 
> **Object** "Types of states of a resource in terms of its stage in a lifecycle."

**Remove:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential are ongoing.

**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Resource is active, current, ongoing, offered, operational, or available.

**Remove:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential are ongoing.

**Add:**
> **Subject:** credentialStat:Active
> **Predicate:** rdfs:comment
> **Object:** Resource is active, current, ongoing, offered, operational, or available.

**Remove:**
> **Subject:** credentialStat:Deprecated
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential have ceased.

**Add:**
> **Subject:** credentialStat:Deprecated
> **Predicate:** rdfs:comment
> **Object:** Resource is permanently no longer active, current, ongoing, offered, operational, or available.

**Remove:**
> **Subject:** credentialStat:Probationary
> **Predicate:** rdfs:comment
> **Object:** Credential award is provisional or temporary; frequently pending further assessment of qualification.

**Add:**
> **Subject:** credentialStat:Probationary
> **Predicate:** rdfs:comment
> **Object:** Resource is active, current, ongoing, offered, operational, or available on a provisional or temporary basis; frequently pending further assessment of qualification.

**Remove:**
> **Subject:** credentialStat:Suspended
> **Predicate:** rdfs:comment
> **Object:** Awards of the credential are not available, but may resume.

**Add:**
> **Subject:** credentialStat:Suspended
> **Predicate:** rdfs:comment
> **Object:** Resource is temporarily not active, current, ongoing, offered, operational, or available.

**Comment by siuc-nate** [comment]
What would the end result of that look like?

**Comment by philbarker** [comment]
@siuc-nate you mean instance data?

**Comment by siuc-nate** [comment]
On rereading it, it looks like's mostly revisions to the terms in the credential status concept scheme, and adding a new property so other classes can reference those terms. Is that correct?

**Comment by philbarker** [comment]
Yep. There's nothing that isn't in one or the other of the proposals above.

**Comment by philbarker** [comment]
Summary from the [proposal doc](https://docs.google.com/document/d/1I2xanxkoU-1gBIX_RId6u6j9w9wv0-T3zxIn_8x-z54/edit#) reads:

>  adding a new widely scoped property statusType that may be used with Learning Opportunities; the existing credentialStatusType would be made a sub-property of the new statusType. The Credential Status concept scheme would be adjusted to allow for its use with a broader set of resource types.
...
> changes to the wording of the Credential Status concepts would indicate it may be used for any "resource" not just credentials. An alteration to the description of the scheme would clarify that it relates to the stages of a resource lifecycle.

**Comment by stuartasutton** [comment]
Looks good me.

**Comment by siuc-nate** [comment]
With the exception of the broadMatch/narrowMatch assertions (the schema management system doesn't have that capability), these changes have been made in pending CTDL and noted in the history tracking.
